vendor/bat-native-ads/src/bat/ads/internal/covariates/log_entries/notification_ad_event_unittest.cc

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/covariates/log_entries/notification_ad_event.h"
+
+#include <sstream>
+
+#include "bat/ads/confirmation_type.h"
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+
+// npm run test -- brave_unit_tests --filter=BatAdsFederatedLogEntries*
+
+namespace ads {
+
+std::string GetValueForEventType(
+    const mojom::NotificationAdEventType event_type) {
+  std::stringstream ss;
+  ss << event_type;
+
+  return ss.str();
+}
```

Comment:
should be in an anonymous namespace

Reply:
We could also add to remove code duplication through tests
```
std::string GetExpectedValueForEventType(
    const mojom::NotificationAdEventType event_type) {
  NotificationAdEvent notification_ad_event;
  notification_ad_event.SetEventType(event_type);
  return notification_ad_event.GetValue();
}
```
i.e.

`const std::string expected_value = GetExpectedValueForEventType(...)`